kfs-web/src/main/webapp/WEB-INF/tags/module/purap/puritems.tag

```diff
@@ -379,7 +379,7 @@
 						    attributeEntry="${itemAttributes.itemTypeCode}"
 						    property="document.item[${ctr}].itemTypeCode"
 						    extraReadOnlyProperty="document.item[${ctr}].itemType.itemTypeDescription"
-						    readOnly="${not ( (fullEntryMode and !amendmentEntry) or (amendmentEntry and itemLine.versionNumber == null)) or itemLine.itemTypeCode == 'UNOR' or lockB2BEntry}" tabindexOverride="${tabindexOverrideBase + 0}"/>

```

Comment:
Please explain why are we excluding the 'lockB2BEntry' test from the readOnly - not clear from requirements....